### PR TITLE
feat(anti-ddos): support new resource to manage default protection policy

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:

support new resource to manage default protection policy.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

- Destroying the resource actually sets the field `traffic_pos_id` to 120 Mbps.
- Default protection strategies always have value.

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.

```
make testacc TEST='./huaweicloud/services/acceptance/antiddos' TESTARGS='-run TestAccDefaultProtectionPolicy_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/antiddos -v -run TestAccDefaultProtectionPolicy_basic -timeout 360m -parallel 4
=== RUN   TestAccDefaultProtectionPolicy_basic
=== PAUSE TestAccDefaultProtectionPolicy_basic
=== CONT  TestAccDefaultProtectionPolicy_basic
--- PASS: TestAccDefaultProtectionPolicy_basic (23.53s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/antiddos  23.580s
```

* [X] Documentation updated.
* [X] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: docs/resources/antiddos_default_protection_policy.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this will create new resource.
+
+* `traffic_threshold` - (Required, Int) Specifies the traffic cleaning threshold in Mbps.
+  The value can be 10, 30, 50, 70, 100, 120, 150, 200, 250, 300, 1000 Mbps.
```
Comment (jinyangyang222) [inline docs/resources/antiddos_default_protection_policy.md:31]:
```suggestion
  The value can be `10`, `30`, `50`, `70`, `100`, `120`, `150`, `200`, `250`, `300`, `1,000` Mbps.
```

File: huaweicloud/services/antiddos/resource_huaweicloud_antiddos_default_protection_policy.go [hunk 2]
```diff
@@ -0,0 +1,203 @@
+// Generated by PMS #350
```
[the remark this answers is not in the document]
Comment (jinyangyang222) [reply huaweicloud/services/antiddos/resource_huaweicloud_antiddos_default_protection_policy.go:1]: This is not a format automatically generated by PMS, there is no need to add this comment

File: docs/resources/antiddos_default_protection_policy.md (hunk 1 above)
Comment (deer-hang) [reply docs/resources/antiddos_default_protection_policy.md]: OK

File: huaweicloud/services/antiddos/resource_huaweicloud_antiddos_default_protection_policy.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (deer-hang) [reply huaweicloud/services/antiddos/resource_huaweicloud_antiddos_default_protection_policy.go]: OK

**Pull request merged**